docs/docs/API-Reference/api-reference-api-examples.md

```diff
@@ -0,0 +1,1587 @@
+
+Download an image file for a given flow.
+
+To look up the file name in Langflow, use the `/list` endpoint.
+
+This example downloads the `2024-12-30_15-42-44_image-file.png` file from Langflow to a file named `output-image.png`.
+
+The `--output` flag is optional.
+
+<Tabs>
+  <TabItem value="curl" label="curl" default>
+
+```curl
+curl -X 'GET' \
+  "$LANGFLOW_URL/api/v1/files/images/$FLOW_ID/2024-12-30_15-42-44_image-file.png" \
+  -H 'accept: application/json' \
+  --output output-image.png
+```
+
+  </TabItem>
+  <TabItem value="result" label="Result">
+
+```plain
+Image file content.
+```
+
+  </TabItem>
+</Tabs>
+
+
+### Delete File
+
+Delete a specific file from a flow.
+
+This example deletes the `2024-12-30_15-42-44_image-file.png` file from Langflow.
+
+<Tabs>
+  <TabItem value="curl" label="curl" default>
+
+```curl
+curl -X 'DELETE' \
+  "$LANGFLOW_URL/api/v1/files/delete/$FLOW_ID/2024-12-30_15-42-44_image-file.png" \
+  -H 'accept: application/json'
+```
+
+  </TabItem>
+  <TabItem value="result" label="Result">
+
+```plain
+{
+  "message": "File 2024-12-30_15-42-44_image-file.png deleted successfully"
+}
+```
+
+  </TabItem>
+</Tabs>
+
+## Logs
+
+Retrieve logs for your Langflow flow.
+
+This endpoint requires that log retrieval be enabled in your Langflow application.
+
+To enable log retrieval, include these values in your `.env` file:
+
+```plain
+LANGFLOW_ENABLE_LOG_RETRIEVAL=true
+LANGFLOW_LOG_RETRIEVER_BUFFER_SIZE=10000
+LANGFLOW_LOG_LEVEL=DEBUG
+```
+
+For log retrieval to function, `LANGFLOW_LOG_RETRIEVER_BUFFER_SIZE` needs to be greater than 0. The default value is `10000`.
+
+Start Langflow with this `.env`:
+
+```plain
+uv run langflow run --env-file .env
+```
+
+### Stream Logs
+
+Stream logs in real-time using Server-Sent Events (SSE).
+
+<Tabs>
+  <TabItem value="curl" label="curl" default>
+
+```curl
+curl -X 'GET' \
+  "$LANGFLOW_URL/logs-stream" \
+  -H 'accept: text/event-stream'
+```
+
+  </TabItem>
+  <TabItem value="result" label="Result">
+
+```plain
+keepalive
+
+{"1736355791151": "2025-01-08T12:03:11.151218-0500 DEBUG Building Chat Input\n"}
+
+{"1736355791485": "2025-01-08T12:03:11.485380-0500 DEBUG consumed event add_message-153bcd5d-ef4d-4ece-8cc0-47c6b6a9ef92 (time in queue, 0.0000, client 0.0001)\n"}
+
+{"1736355791499": "2025-01-08T12:03:11.499704-0500 DEBUG consumed event end_vertex-3d7125cd-7b8a-44eb-9113-ed5b785e3cf3 (time in queue, 0.0056, client 0.0047)\n"}
+
+{"1736355791502": "2025-01-08T12:03:11.502510-0500 DEBUG consumed event end-40d0b363-5618-4a23-bbae-487cd0b9594d (time in queue, 0.0001, client 0.0004)\n"}
+
+{"1736355791513": "2025-01-08T12:03:11.513097-0500 DEBUG Logged vertex build: 729ff2f8-6b01-48c8-9ad0-3743c2af9e8a\n"}
+
+{"1736355791834": "2025-01-08T12:03:11.834982-0500 DEBUG Telemetry data sent successfully.\n"}
+
+{"1736355791941": "2025-01-08T12:03:11.941840-0500 DEBUG Telemetry data sent successfully.\n"}
+
+keepalive
+```
+
+  </TabItem>
+</Tabs>
+
+### Logs
```

Comment:
On line 1294, you have a level 2 heading called "Logs". I realize this one is a level 3 and has different information than the other heading, but it looks like it's a mistake. Can you name this one something else? Maybe "Logs with optional parameters"?

Reply:
Good call. Updated with more information on the parameters.